GameData/RealismOverhaul/Engine_Configs/TD339_Config.cfg

```diff
@@ -110,17 +111,17 @@
 }
 
 //21 engines, 0 failed
-//63 ignitions, 1 failed
+//63 ignitions, 1 failed  --  Source labels 0 flight failures, 100% flight reliability
 @PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[TD-339]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
 {
 	TESTFLIGHT
 	{
 		name = TD-339
 		ratedBurnTime = 495		//Enough to deplete fuel tanks at low throttle
-		ignitionReliabilityStart = 0.984127
-		ignitionReliabilityEnd = 0.996825
-		cycleReliabilityStart = 0.954545
-		cycleReliabilityEnd = 0.990909
+		ignitionReliabilityStart = 98.4
+		ignitionReliabilityEnd = 99.6
+		cycleReliabilityStart = 95.4
+		cycleReliabilityEnd = 99.1
```

Comment:
Revert this please

Reply:
Whoops. That was my bad. Wrote them down in my change log as percentages, and when I reverted that I forgot to change back to a decimal. will fix.